pytorch_blade/.bazelrc

```diff
@@ -1,8 +1,9 @@
 try-import ../tf_community/.bazelrc
 try-import ../tf_community/.bazelrc.user
 
+build --disk_cache=~/.cache
```

Comment:
well, this is to use local disk as 'remote cache' ... so the true 'remote-cache' is still disabled IMO.